monkey-runner/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Tue Oct 20 10:41:37 BST 2015
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.13-all.zip
```

Comment:
Latest version is [2.14.1](https://docs.gradle.org/current/release-notes). Have you tried using it?